src/net/sourceforge/kolmafia/request/GenericRequest.java

```diff
@@ -2158,6 +2158,12 @@ public void processResponse() {
       GenericRequest.isRatQuest = false;
     }
 
+    // Check that we are actually handling a choice
+    if (ChoiceManager.handlingChoice
+        && this.responseText.contains("Whoops!  You're not actually in a choice adventure.")) {
+      return;
```

Comment:
OK. I guess it is fair to short-circuit the processing; there is nothing more of interest to process in the response.

However, by doing so, you are not letting postChoice2 set ChoiceManager.handlingChoice to false.

You could do that here, although every place that sets that variable is currently in ChoiceManager.

I dunno. Perhaps:
```
    
    if (ChoiceManager.bogusChoice(this)) {
      return;
    }
```
And something like:
```
  public static final boolean bogusChoice(final GenericRequest request) {
    if (!ChoiceManager.handlingChoice) {
      return false;
    }
    if (request.responseText.contains("Whoops!  You're not actually in a choice adventure.") {
      ChoiceManager.handlingChoice = false;
      return true;
    }
    return false;
  }
```

Reply:
Ah I was previously doing that and removed it without thinking